mockdata/api/clusters_mgmt/v1/clusters.json

```diff
@@ -3232,6 +3232,254 @@
       "etcd_encryption": false,
       "billing_model": "standard"
     },
+    {
```

Comment:
please re-run the script to re-generate this file once all cluster data is updated.